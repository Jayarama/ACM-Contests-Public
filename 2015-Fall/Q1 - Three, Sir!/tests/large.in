60 5 12
36 12 5
45 60 17
40 5 37
60 5 12
12 5 7
40 15 600
35 140 4
37 15 52
37 15 52
36 12 5
40 5 37
36 12 5
60 5 12
45 60 17
12 5 7
40 15 600
37 15 52
40 5 37
12 5 7
40 15 600
45 60 17
12 5 7
40 15 600
36 12 5
35 140 4
60 5 12
45 60 17
37 15 52
35 140 4
35 140 4
40 5 37
