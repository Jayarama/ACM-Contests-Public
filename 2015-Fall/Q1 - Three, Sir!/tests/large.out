Carry on.
Three, Sir!
Three, Sir!
Three, Sir!
Carry on.
Carry on.
Carry on.
Three, Sir!
Carry on.
Carry on.
Three, Sir!
Three, Sir!
Three, Sir!
Carry on.
Three, Sir!
Carry on.
Carry on.
Carry on.
Three, Sir!
Carry on.
Carry on.
Three, Sir!
Carry on.
Carry on.
Three, Sir!
Three, Sir!
Carry on.
Three, Sir!
Carry on.
Three, Sir!
Three, Sir!
Three, Sir!
